src/yunohost/data_migrations/0003_migrate_to_stretch.py

```diff
@@ -29,11 +29,7 @@ class MyMigration(Migration):
 
     mode = "manual"
 
-    def backward(self):
-
-        raise YunohostError("migration_backward_impossible", name=self.name)
-
-    def migrate(self):
+    def forward(self):
```

Comment:
You've renamed everything to `forward` (which is not really part of this commit if we look at the commit message :/) but [it's still the `migrate` method that is called](https://github.com/YunoHost/yunohost/pull/768/files#diff-d1fe4f9d57b5045a5baa68a74c263423R1150) so this will break everything :zipper_mouth_face:

Reply:
I have renamed it just because some migration use forward and other migrate.

Migrate method is marked as to not override in the class Migration. this migrate method call forward, and i didn't find other part of the code calling this method. But i am currently doing an other pr derivated of this one with other stuff.